engine/src/main/java/org/terasology/config/BindsConfig.java

```diff
@@ -133,15 +129,15 @@ public BindsConfig deserialize(JsonElement json, Type typeOfT, JsonDeserializati
         public JsonElement serialize(BindsConfig src, Type typeOfSrc, JsonSerializationContext context) {
             JsonObject result = new JsonObject();
             SetMultimap<Name, SimpleUri> bindByModule = HashMultimap.create();
-            for (SimpleUri key : src.data.keySet()) {
+            for (SimpleUri key : src.bindUriToInput.keySet()) {
                 bindByModule.put(key.getModuleName(), key);
             }
             List<Name> sortedModules = Lists.newArrayList(bindByModule.keySet());
             Collections.sort(sortedModules);
             for (Name moduleId : sortedModules) {
                 SetMultimap<String, Input> moduleBinds = HashMultimap.create();
                 for (SimpleUri bindUri : bindByModule.get(moduleId)) {
-                    moduleBinds.putAll(bindUri.getObjectName().toString(), src.data.get(bindUri));
+                    moduleBinds.putAll(bindUri.getObjectName().toString(), src.bindUriToInput.get(bindUri));
```

Comment:
Similar concern here with the naming of variables and verbs vs nouns: I'd rename bindUri to bindingUri.